pkg/lightning/errormanager/errormanager.go

```diff
@@ -269,6 +281,14 @@ func (em *ErrorManager) Init(ctx context.Context) error {
 		}
 	}
 
+	if em.conflictV1Enabled && em.conflictV2Enabled {
```

Comment:
should we always tell the user one VIEW to find the duplicate error, no matter how lightning is configurated?

Reply:
I think it is better that we only use VIEW when there are data in both tables. For the cases where the table contain a huge amount of data, using VIEW may slow down the performance when the result is exactly the same with not using VIEW.
It will be clearly stated in the log which table or view to check the conflict data.

Reply:
this will increase the complexity to user, also check it with PM. And for the performance problem I think 10000 records will not change response time a lot, you can test it afterwards.

Reply:
PM said the current implementation is acceptable, but it is better that we use VIEW to return to users regardless of the configuration in the later optimization. Will revise this in the next PR